.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+https://discord.gg/ZeuASSx.
```

Comment:
```suggestion
reported to the community leaders responsible for enforcement via
their emails as accessed via their github profile or as listed in the
Endless Sky repository, such as the copyright and commit logs.
```

This is a GitHub community code of conduct. It should not require people to join another 3rd party platform just to file a report. Specific emails could be listed here, or directions to specifically where to find them. Either would be considerably more useful than linking to a discord server.

Reply:
> My immediate thought is that some sort of Google form would be a good option. As far as I know, you don't even need a Google account to submit a response to a Google form (unless you wish to include uploaded files).

This is correct; yes, assuming you don't need to upload files, a Google form can be submitted anonymously. 

Another advantage to a Google form is that the form submissions can be shared among a group of people, so anyone whose been added to the form can see responses to it, whereas with email or Discord (if they choose to DM), whichever individual receives the email/DM could ignore it if they don't feel it's worth acting upon. In contrast, with a Google form, at the very least, a team of people could be made aware of it, making things much less likely to get ignored.

Reply:
I rank email considerably above Discord because Discord is a singular entity (as is google); whereas there are millions of email providers (or even self-hosted) to suit every whim. 

A google form would definitely be much better than the discord link. That being said, just straight up listing everyone who can take appropriate action here with their particular preferred method of contact would be reasonable, too.

The downside with the google form is "what if the person is reporting a problem with someone who has access to the place where that google form goes?"

Reply:
> I rank email considerably above Discord because Discord is a singular entity (as is google); whereas there are millions of email providers (or even self-hosted) to suit every whim.

I'm not sure why this makes Dsicord or Google forms an undesirable means of communication here. Would you elaborate?

Reply:
Email is not a third party service but an internet protocol, this is an important difference, at least to people like me who value privacy and maybe don't want to use services provided by big companies (and only these companies) such as google or discord.

This is not to say those should not be listed, but in no way should those be the only ones listed.

Reply:
RisingLeaf nailed it. Discord has a monopoly on itself. If someone objects to something in the ToS, then they are barred from using it at all. Blocked out entirely. If someone doesn't like google's ToS, they can go use some other provider's email service. Zero bar to entry. Take facebook, as an example. Lots of groups on there, but one has to register with a real name, and often provide a driver's license or phone number connected to name in order to register. If someone objects to that, then they're locked out. There's no recourse or other option.

Circling back to the topic at hand, this is a GitHub code of Conduct, primarily applicable to actions and communication here. GitHub already requires an email, so that is likely something that everyone active here already has and is familiar with. It's known technology for people active here. If they want to create a new email for communication here's purposes, they are capable of doing so, or not, as they prefer.

As such, email has the lowest bar to entry for the situation at hand, and has the least problems in terms of privacy and 3rd party restrictions.

Reply:
Naev gives the option of reporting violations via Discord and email.

I don't see why we can't have both an email and a google form. That seems like those two combined would cover most people. There should be a warning if using a spam filter that email addresses from less common providers may be sent to spam by the server, because that does happen, but other than that I don't see the issue of giving multiple options.

Reply:
Setting up a collaborative inbox with Google Groups isn't too hard, though there's a bit of structural overhead.
If people find that desirable I can set one up.

Reply:
> Setting up a collaborative inbox with Google Groups isn't too hard, though there's a bit of structural overhead. If people find that desirable I can set one up.

A shared email inbox, alongside a Google form shared with the same group of individuals, would be the ideal solution, assuming that the shared inbox works how I envision it would.
That solution would allow an individual to use email to avoid using a third-party service or submit an anonymous form response if they aren't comfortable sharing their email address.
It also avoids the drawbacks of relying on a single point of contact as the only person responsible for receiving, communicating, and acting upon complaints. 

Do you happen to know if a shared inbox would allow individuals with access to that shared inbox to modify/delete incoming complaints in such a way that it wouldn't be traceable by the other individuals with access to that inbox?

Reply:
Depends on the shared inbox I think. For [Microsoft Outlook](https://support.microsoft.com/en-us/office/open-and-use-a-shared-mailbox-in-outlook-d94a8e9e-21f1-4240-808b-de9c9c088afd), it seems like only those who could delete incoming emails would be admins of Microsoft's server. However, for [Gmail](https://support.google.com/googlecloud/answer/10636599?hl=en), "delegates" would be able to delete incoming emails, so mods would have to have something like POP3 set up to download emails as they arrive, so that they would have a local copy in case that happens.